README.rst

```diff
@@ -313,6 +313,36 @@ The ``version_id`` transport parameter enables you to get the desired version of
   ...     print(repr(fin.read()))
   'second version\n'
 
+GCS Credentials
+---------------
+``smart_open`` uses the ``google-cloud-storage`` library to talk to GCS.
+``google-cloud-storage`` uses the ``google-cloud`` package under the hood to handle authentication.
+There are several `options <https://google-cloud-python.readthedocs.io/en/0.32.0/core/auth.html>`__ to provide
+credentials.
+By default, ``smart_open`` will defer to ``google-cloud-storage`` and let it take care of the credentials.
+
+To override this behavior, pass a ``google.cloud.storage.Client`` object as a transport parameter to the ``open`` function.
+You can `customize the credentials <https://google-cloud-python.readthedocs.io/en/0.32.0/core/client.html>`__
+when constructing the client. ``smart_open`` will then use the client when talking to GCS.
+
+.. code-block:: python
+
+    >>> from google.cloud.storage import Client
+    >>> client = Client.from_service_account_json('/path/to/keyfile.json')
+    >>> fin = open('gs://bucket/key', transport_params=dict(client=client), ...)
```

Comment:
Same thing with the URL. Ideally, it should be real. Is it difficult to set up a publicly readable GCS bucket and key?

Reply:
I ended up using one of Google's [public datasets](https://cloud.google.com/storage/docs/public-datasets/landsat) instead